surveys/2021-annual-survey/questions.md

```diff
@@ -366,19 +366,27 @@ Select all that apply:
 - From source
 - Other
 
-### If you don't use rustup, why not?
 
-Select all that apply:
+### In your opinion, how do you find the following aspects of Rust?
 
-- I'm prefer using my platform's package installer (yum, apt-get, homebrew, msi installer, etc.)
-- I don't trust that rustup provides secure binaries
-- I didn't know about it
-- The network I'm on does not allow for it (e.g., firewall)
-- I tried, but it didn't work
-- Other
+Options:
 
-> TODO: a lot of people previously reported that they *use* rustup but wish they didn't. How can we capture that here?
-> TODO: this was previously free form. Do we prefer the multiple choice?
+- Great
+- Good enough
+- Could be better
+- Terrible
```

Comment:
Nit: Thinking more about it, I'd prefer to not normalize the use of "terrible" to describe things. Perhaps "in need of serious improvement"?